# case3

try:
    ...
    
except 발생 오류 as 오류 메시지 변수:
    ...